march_hardware/src/Joint.cpp

```diff
@@ -140,7 +140,7 @@ IMotionCubeState Joint::getIMotionCubeState()
   states.motionErrorDescription = error::parseMotionError(this->iMotionCube.getMotionError());
 
   states.motorCurrent = this->iMotionCube.getMotorCurrent();
-  states.motorVoltage = this->iMotionCube.getMotorVoltage();
+  states.IMCVoltage = this->iMotionCube.getIMCVoltage();
```

Comment:
As stated in my other comment, I believe that you are using an SDO object which does not support PDO mapping and therefore cannot make use of the same get-function. In PM-381 I will provide read functions for sdo objects, which should allow you to make use of 2065h and 2066h to read the motor voltage appropriately.

Reply:
This is a renaming of the old method to read 0x2055, the IMC's input voltage. Cilia will probably add a getter for the motor voltage in another PR :)

Reply:
So the function getIMCVoltage does not yet exist?

Reply:
No it does. getMotorVoltage no longer exists because she renamed it to getIMCVoltage

Reply:
Maybe it's good to clear some things up. I already made this branch to do add the Motor Voltage, but I realized that there were more steps. That's why I decided to make smaller pull requests, to make it easier and faster for you to comment my work (and for me to get trough the feedback). But I only have one Jira issue and one branch, so I did it this way and plan to continue to work on this Branch. I am not sure if this is they right way to do it, so if you guys think there is a better way to do this I like to hear it